chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -893,8 +893,8 @@ def create_zpools(self):
         fqdns = [x['fqdn'] for x in xs]
 
         self.execute_simultaneous_commands(
-            ['modprobe zfs'], fqdns,
-            'checking for zfs presence',
+            ['modprobe zfs && systemctl start zfs-zed'], fqdns,
```

Comment:
This should be removed (both the modprobe and starting ZED). Underlying cause is being fixed in https://github.com/intel-hpdd/iml-common